packages/public/client/src/main/index.js

```diff
@@ -199,7 +199,8 @@ const init = $context => {
       initEntityEditor(
         {
           initialState: $editor.data('state'),
-          type: $editor.data('type')
+          type: $editor.data('type'),
+          mayCheckout: $editor.data('may-checkout')
```

Comment:
Is this really a boolean?

Reply:
well.... it's a 1 or 0 atm? :sweat_smile: